presto-plugin-toolkit/src/test/java/io/prestosql/plugin/base/security/TestFileBasedSystemAccessControl.java

```diff
@@ -487,6 +488,19 @@ public void testTableRulesForCheckCanRenameColumn()
         assertAccessDenied(() -> accessControl.checkCanRenameColumn(BOB, new CatalogSchemaTableName("some-catalog", "bobschema", "bobtable")), RENAME_COLUMNS_ACCESS_DENIED_MESSAGE);
     }
 
+    @Test
+    public void testTableRulesForCheckCanSetTableAuthorization()
```

Comment:
I wonder whether it wouldn't be better to have 3 tests here... with proper names of the test cases that they are checking.

Reply:
The test is made according to how the class is written. 
`testTableRules` means we are testing multiple rules. Is it TDD or any other testing acronym compliant - probably not. Just like the whole Presto